### ISSUE TITLE
Add hypothesis tests for all test functions

### ISSUE THREAD

**Issue by OriolAbril** [arviz-devs/arviz-plots#63]
Tests with hypothesis do a very good job of making sure the API works and is coherent. Today for example I realized about #62 which would have been caught by these tests.

**Comment by OriolAbril** [comment]
reopening because the PR that closed this automatically fixed this for `plot_compare` but there are still missing ones

**Comment by PiyushPanwarFST** [comment]
Yes, I will start adding hypothesis tests for the remaining functions, starting with `combine_plots`
